src/NzbDrone.SignalR/NzbDronePersistentConnection.cs

```diff
@@ -31,17 +31,6 @@ public NzbDronePersistentConnection(IConfigFileProvider configFileProvider)
 
         public void BroadcastMessage(SignalRMessage message)
         {
-            string lastMessage;
-            if (_messageHistory.TryGetValue(message.Name, out lastMessage))
-            {
-                if (message.Action == ModelAction.Updated && message.Body.ToJson() == lastMessage)
-                {
-                    return;
-                }
-            }
-
-            _messageHistory[message.Name] = message.Body.ToJson();
-
```

Comment:
What was this removed for?

Reply:
Good spot - I'll put it back.  I removed this when I was tracking down a bug that turned out to be caused by `ToJson` evaluating all the `LazyLoaded` instances causing an infinite loop.